### ISSUE TITLE
Issue with "The box model": ul inherits display: flex, thus the example is misleading

### ISSUE THREAD

**Issue by martenmatrix** [mdn/content#8780]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model

#### What information was incorrect, unhelpful, or incomplete?
The ul element, in the example, inherits a `display: flex;` attribute. Due do this the example is misleading.

The instructions state _"Add display: inline-block to the rule with the .links-list a selector, and you will see how it fixes this issue by causing the padding to be respected by other elements."_ . This is true, but in that specific example the user also could just set the display-property to `display: block;` as the ul element inherits a display: flex; property. The display: flex; property prevents the block element from creating a new line, I believe.

#### Specific section or headline?
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model#using_display_inline-block - SECOND EXAMPLE

#### What did you expect to see?
A new created line for every list entry, when I set the display-property of the class _.links-list a_ to block;

#### Did you test this? If so, how?
I set the display property of ul to block, and it worked as expected.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/css/building_blocks/the_box_model`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/css/building_blocks/the_box_model/index.html
* Last commit: https://github.com/mdn/content/commit/b7c594e7a8e5eb31a70bcf3e07d931e4603d201b
* Document last modified: 2021-09-03T05:34:55.000Z

</details>

**Comment by rachelandrew** [comment]
It's not possible for an element to inherit `display: flex`, the ul is a flex container by design. However, it sounds as if you expect this example to do something it isn't doing.

The aim of the example is for the padding to be respected, not to put the items onto new lines. 

You are asked to fix the following issue:

> "The padding appears to overlap the border on the <ul> element. This is because the <a> is an inline element."

**Comment by martenmatrix** [comment]
Oh, I didn't know that. I just found it misleading, that `display: block;` and `display: inline-block;` putted out the same result until I found out that the ul is a flex container by design. Because the example states that _`display: block-inline` is useful in the following example_(second one), which it isn't, as I could just use `display: block`. I found it misleading when learning, but maybe that's just my point of view. :)
